ERROR: "IndexError" exception when attempting to generate a report for {'intrcn_only_gene': 'YWHAZ', 'fusion_only_gene': 'FAM22B', 'YWHAZ': 'P63104', 'YWHAE': 'P62258', 'intrcn_and_fusion_gene': 'YWHAE'}: list index out of range